zone/client.cpp

```diff
@@ -606,10 +607,118 @@ void Client::RemoveExpendedAA(int aa_id)
 	database.QueryDatabase(StringFormat("DELETE from `character_alternate_abilities` WHERE `id` = %d and `aa_id` = %d", character_id, aa_id));
 }
 
+//bool Client::Save(uint8 iCommitNow) {
```

Comment:
Don't do this. This code is saved in the version control software :P

Reply:
Ahh I left that there for ease of changing back and forth between the old/new. I was assuming before this would even be accepted I would do a full cleanup. First experience of a PR to this project.